frontend/pages/hosts/ManageHostsPage/HostTableConfig.tsx

```diff
@@ -51,24 +54,48 @@ interface ICellProps {
   };
 }
 
-interface IHostDataColumn {
-  Header: ((props: IHeaderProps) => JSX.Element) | string;
-  Cell: (props: ICellProps) => JSX.Element;
-  id?: string;
-  title?: string;
-  accessor?: string;
-  disableHidden?: boolean;
-  disableSortBy?: boolean;
+interface IDeviceUserCellProps {
+  cell: {
+    value: IDeviceUser[];
+  };
+  row: {
+    original: IHost;
+  };
 }
 
+const condenseDeviceUsers = (users: IDeviceUser[]): string[] => {
+  if (!users?.length) {
+    return [];
+  }
+  const condensed =
+    users
+      .slice(-3)
+      .map((u) => u.email)
+      .reverse() || [];
+  return users.length > 3
+    ? condensed.concat(`+${users.length - 3} more`) // TODO: confirm limit
+    : condensed;
+};
+
+const tooltipTextWithLineBreaks = (lines: string[]) => {
+  return lines.map((line) => {
+    return (
+      <span key={Math.random().toString().slice(2)}>
```

Comment:
what do you think of using `line` as the key? it would avoid re-rendering this all the time and shouldn't be costly if they are just short strings

Reply:
It's a very good thought. I tested both approaches just now and they render the same number of times.  

I considered just using the line string at first then opted for this other approach because I didn't have guarantees that the line strings would be unique. Of course, there isn't an absolute guarantee that the pseudo-random function would produce unique keys unique either. In some places our code is even using uuidv4 values for these sorts of keys. I thought that was probably overkill and wanted to try out this approach after benchmarking uuid vs. the above approach.